### PR TITLE
Add label synonym polysemy check

### PR DESCRIPTION
**Pull request by anitacaron**
This is the same check available in [Uberon](https://github.com/obophenotype/uberon/blob/master/src/sparql/label-synonym-polysemy-violation.sparql).

Currently, this check has two violations as already raised in the issue 
- #2101:

```
FAIL Rule ../sparql/label-synonym-polysemy-violation.sparql: 2 violation(s)
entity,property,value
http://purl.obolibrary.org/obo/CL_0002294,http://www.geneontology.org/formats/oboInOwl#hasExactSynonym,subcapsular thymic epithelial cell
http://purl.obolibrary.org/obo/CL_0009077,http://www.w3.org/2000/01/rdf-schema#label,subcapsular thymic epithelial cell
```

### CODE REVIEW

**Comment by anitacaron** [comment]
The two violations need to be fixed for this PR can be merged.

**Comment by bvarner-ebi** [comment]
@anitacaron, #2101 has been resolved. Can you proceed with this PR or is it also dependent on #2102?

**Comment by anitacaron** [comment]
@bvarner-ebi, this doesn't depend on #2102 because it's already on the ROBOT report as WARN. 

Now, after updating the branch from master, it just need to pass the checks and it can be merged.

File: src/sparql/label-synonym-polysemy-violation.sparql [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        oboInOwl:hasExactSynonym
+        rdfs:label
+      }
+      ?entity ?property ?name .
+      OPTIONAL {
+        ?abbr a owl:Axiom;
+              owl:annotatedSource ?entity;
+              owl:annotatedProperty oboInOwl:hasExactSynonym;
+              owl:annotatedTarget ?name;
+              oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/uberon/core#ABBREVIATION>.
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply src/sparql/label-synonym-polysemy-violation.sparql:24]: I'll update to this annotation that is the only one used here in CL.

**Review by bvarner-ebi — APPROVED** [review]
Tested locally.
Expected output observed.
Thanks, @anitacaron!

**Pull request merged**